controls/cis_rhel8.yml

```diff
@@ -774,26 +774,25 @@ controls:
     related_rules:
       - service_avahi-daemon_disabled
 
-  - id: 2.2.4
-    title: Ensure CUPS is not installed (Automated)
+  - id: 2.2.3
+    title: Ensure dhcp server services are not in use (Automated)
     levels:
       - l1_server
+      - l1_workstation
     status: automated
     rules:
-      - package_cups_removed
+      - package_dhcp_removed
     related_rules:
-      - service_cups_disabled
-
-  - id: 2.2.5
-    title: Ensure DHCP Server is not installed (Automated)
+      - service_dhcpd_disabled
```

Comment:
Could you also update the reference in "service_dhcpd_disabled", please? It shows 2.2.5.